src/css/fontFamilies.scss

```diff
@@ -0,0 +1,48 @@
+@font-face {
```

Comment:
In netlify says that you need to add `@charset "UTF-8";` please check

![image](https://user-images.githubusercontent.com/36519478/178094460-75cf4c16-df2d-4666-805f-eec839f41c1b.png)

Reply:
I haven't used it before, I will try